draft-issuance.md

```diff
@@ -0,0 +1,176 @@
+
+In summary, by introducing a smoother emissions curve, we:
+- maintain the economic viability of Zcash
+- provide clarity on distribution of deposits into the ZSF
+- enhance Zcash's stability as the network evolves.
+
+# Requirements
+
+Smoothing the issuance curve is possible using an exponential decay formula that
+satisfies the following requirements:
+
+## Issuance Requirements
+
+1. The issuance can be summarised into a reasonably simple explanation
+2. Block subsidies approximate a continuous function
+3. If there are funds in the ZSF, then the block subsidy must be non-zero, preventing any final “unmined” zatoshis
+4. For any 4 year period, all paid out block subsidies are approximately equal to half of the ZSF at the beginning of that 4 year period, if there are no deposits into the ZSF during those 4 years
+TODO daira: add a requirement that makes the initial total issuance match the previous total issuance  
+5. This functionality MUST be introduced as part of a network upgrade
+
+
+# Specification
+
+## Constants
+
+Define constants:
+
+“`BLOCK_SUBSIDY_FRACTION`” = 41 / 100,000,000 or `0.00000041`
+
+"`DEPLOYMENT_BLOCK_HEIGHT`" = 2726400
+
+## Issuance Calculation
+
+At the `DEPLOYMENT_BLOCK_HEIGHT`, nodes should switch from the current issuance calculation, to the following:
+
+Given the block height `h` define a function **BlockSubsidy(h)**, such that:
+
+**BlockSubsidy(h)** = Block subsidy for a given `h`, that satisfies above requirements.
+
+Using an exponential decay function for **BlockSubsidy** satisfies requirements **R1** and **R2** above:
+
+`BlockSubsidy(h) = BLOCK_SUBSIDY_FRACTION * ZsfBalanceAfter(h - 1)`
+
+Finally, to satisfy **R3** above we always round up to the next zatoshi.
+
+`BlockSubsidy(h) = ceiling(BLOCK_SUBSIDY_FRACTION * ZsfBalanceAfter(h - 1))`
+
+## Deployment
+
+# Rationale
+
+## `BLOCK_SUBSIDY_FRACTION`
+
+Let `IntendedZSFFractionRemainingAfterFourYears` = 0.5.
+
+The value `41 / 100_000_000` satisfies the approximation:
+
+`(1 - BLOCK_SUBSIDY_FRACTION)^PostBlossomHalvingInterval ≈ IntendedZSFFractionRemainingAfterFourYears`
+
+Meaning after a period of 4 years around half of `ZSF_BALANCE` will be paid out
+as block subsidies, thus satisfying **R4**.
+
+## `DEPLOYMENT_BLOCK_HEIGHT`
+
+The deployment should happen at the next halving, which is block `2726400`.
+
+Since there is a planned halving at this point, there will already be a significant "shock" caused by the drop in issuance caused by the halving. This reduces surprise and thus increases security. Also, due to the nature of the smoothed curve having a portion of the curve above the respective step function line at times, this will maximally _reduce_ the issuance shock at the `TARGET_BLOCK_HEIGHT`.
```

Comment:
It looks like `DEPLOYMENT_BLOCK_HEIGHT` and `TARGET_BLOCK_HEIGHT` might be the same constant.

```suggestion
Since there is a planned halving at this point, there will already be a significant "shock" caused by the drop in issuance caused by the halving. This reduces surprise and thus increases security. Also, due to the nature of the smoothed curve having a portion of the curve above the respective step function line at times, this will maximally _reduce_ the issuance shock at the `DEPLOYMENT_BLOCK_HEIGHT`.
```

Reply:
IMO, it is worth noting in the ZIP text here that while this height will reduce the drop in issuance vs the halving schedule (so that issuance rate will be more than half of the previous amount) it will lead to lower issuance than the halving schedule at some time in the future (assuming ZSF deposits are not large enough to shift this).

In other words it lessons the shock/disruption in the immediate term and a cost in the future. However that future cost isn't sudden and gradual.